src/IBM.Cloud.SDK.Core/Service/IBMService.cs

```diff
@@ -19,22 +19,18 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using IBM.Cloud.SDK.Core.Authentication;
-using IBM.Cloud.SDK.Core.Authentication.BasicAuth;
-using IBM.Cloud.SDK.Core.Authentication.Iam;
-using IBM.Cloud.SDK.Core.Authentication.Icp4d;
+using IBM.Cloud.SDK.Core.Authentication.Cp4d;
 using IBM.Cloud.SDK.Core.Authentication.Noauth;
 using IBM.Cloud.SDK.Core.Http;
 using IBM.Cloud.SDK.Core.Util;
-
+
 namespace IBM.Cloud.SDK.Core.Service
 {
     public abstract class IBMService : IIBMService
     {
-        //const string PATH_AUTHORIZATION_V1_TOKEN = "/authorization/api/v1/token";
         private const string icpPrefix = "icp-";
         private const string apikeyAsUsername = "apikey";
-        public string SERVICE_NAME;
-        public string ApiKey { get; set; }
+        public string serviceName;
         public IClient Client { get; set; }
         private bool skipAuthentication = false;
```

Comment:
I think we should just rely on the use of `NoauthAuthenticator` rather than keeping this flag as well